framework/sessions/__init__.py

```diff
@@ -88,9 +88,7 @@ def get_session():
     secret = settings.SECRET_KEY
     cookie = request.cookies.get(settings.COOKIE_NAME)
     if not cookie:
-        s = SessionStore()
-        s.create()
-        return s
+        return SessionStore()
```

Comment:
Locally verified that each non-v2 request will create a session when there is no cookie. (e.g. assets)

```
if not cookie:
    s = SessionStore()
    print(f'>>>> INFO: get_session() called without a cookie, [s.session_key={s.session_key}]')
    return s
```

In addition, `SessionStore()` doesn't create a session key until the session is `.saved()`. Thus we may need to be careful in other places where we use the object returned from `get_session()`.

![Screen Shot 2023-03-09 at 6 06 56 AM](https://user-images.githubusercontent.com/3750414/224005972-d64f0227-3afc-478b-b1df-85e75dd89134.png)
![Screen Shot 2023-03-09 at 6 07 40 AM](https://user-images.githubusercontent.com/3750414/224005973-0713d1b8-3c59-442a-8e93-6ba83a087e9c.png)

Reply:
In addition, it is ok to not save all these sessions (even when the session is not created from asset requests). The argument is: we always save a session when we store stuff in it, that's when the `session_key` is generated and when the session is stored in the Redis.